element_deeplabcut/readers/dlc_reader.py

```diff
@@ -157,16 +160,20 @@ def do_pose_estimation(video_filepaths, dlc_model, project_path, output_dir,
     # Write dlc config file to base (data) folder
     # This is important for parsing the DLC in datajoint imaging
     output_dir.mkdir(exist_ok=True)
-    dlc_cfg_filepath = output_dir / 'dlc_config_file.yaml'
+    dlc_cfg_filepath = dlc_project_path / 'dlc_config_file.yaml'
```

Comment:
Where is `dlc_project_path` coming from? I'm assuming you mean `project_path` from the input argument? If so, why `project_path`, the intention is to store the pose estimation results at the directory specified by `output_dir`, right?

Reply:
`dlc_project_path` is defined in [L150](https://github.com/ttngu207/element-deeplabcut/blob/main/element_deeplabcut/readers/dlc_reader.py#L150) using the `project_path` yes. 

Do we ever read this file? Is it just metadata to store with the output? It's not clear to me why we're keeping [only this restricted set](https://github.com/ttngu207/element-deeplabcut/blob/main/element_deeplabcut/dlc.py#L349-L351)